repo/blob/azure/azure_storage.go

```diff
@@ -497,6 +509,31 @@ func getOriginalFile(deleteMarkerFile string) (originalFile string, ok bool) {
 func New(ctx context.Context, opt *Options, isCreate bool) (blob.Storage, error) {
 	_ = isCreate
 
+	raw, err := newStorage(opt)
+	if err != nil {
+		return nil, err
+	}
+
+	st, err := maybePointInTimeStore(ctx, raw, opt.PointInTime)
```

Comment:
This is the only change here. The rest is because I extracted part into `newStorage` like on s3